examples/03-advanced/02-volume_averaged_stress.py

```diff
@@ -11,7 +12,7 @@
 
 .. note::
     This example requires the Premium ServerContext.
-    For more information, see :ref:`_ref_getting_started_contexts`.
+    For more information, see :ref:`_user_guide_server_context`.
```

Comment:
```suggestion
    For more information, see :ref:`user_guide_server_context`.
```

Reply:
Thanks, I was looking at how to fix that, did not guess that one !